### DISCUSSION TITLE
Object Equality examples in "Programming in Scala 5th edition" by Odersky, Spoon, Venners, Sommers

### DISCUSSION THREAD

**Discussion by spamegg1** [scala/scala3#13993] [General Discussion]
I have Scala 3.1 that I got from Homebrew: `brew install lampepfl/brew/dotty` and OpenJDK 11:
```bash
 ➜ scala --version
Scala code runner version 3.1.0 -- Copyright 2002-2021, LAMP/EPFL
 ➜ java --version
openjdk 11.0.11 2021-04-20
OpenJDK Runtime Environment (build 11.0.11+9-Ubuntu-0ubuntu2.20.04)
OpenJDK 64-Bit Server VM (build 11.0.11+9-Ubuntu-0ubuntu2.20.04, mixed mode, sharing)
```

I have the June 14, 2021 version of the book. On Page 123, Section 5.8 it says

Going further, you can compare two objects that have different types:
```scala
1 == 1.0 // true: Boolean
List(1, 2, 3) == "hello" // false: Boolean
```
However my REPL says:
```scala
 ➜ scala
Welcome to Scala 3.1.0 (11.0.11, Java OpenJDK 64-Bit Server VM).
Type in expressions for evaluation. Or try :help.
                                                                                                                              
scala> List(1,2,3) == "hello"
-- Error:
1 |List(1,2,3) == "hello"
  |^^^^^^^^^^^^^^^^^^^^^^
  |Values of types List[Int] and String cannot be compared with == or !=
```
This is not a compiler/REPL bug, right? This looks in line with [Multiversal Equality](https://docs.scala-lang.org/scala3/book/ca-multiversal-equality.html) 

So, the book must be in error? Or maybe things changed from 3.0 to 3.1? I submitted to [Errata](https://booksites.artima.com/programming_in_scala_5ed/errata) but just wanted to double-check.

**Comment by odersky** [comment]
That looks indeed like an error in the book. Can you send an erratum to Bill Venners? Thanks!

**Reply by spamegg1** [reply]
Done, thanks for the quick response Martin!
